starter_code/queries.md

```diff
@@ -4,80 +4,98 @@
 
 ### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+<!-- {name: { $eq: "Babelgum"}} -->
```

Comment:
To retrieve only the 'name' field, you can add : 
Projection: {name: 1, _id: 0}